charts/galley/values.yaml

```diff
@@ -30,6 +30,9 @@ config:
       sso: disabled-by-default
       legalhold: disabled-by-default
       teamSearchVisibility: disabled-by-default
+      classifiedDomains:
+        status: disabled
+        config: {}
```

Comment:
This is an invalid configuration:

```
galley: user error (AesonException "Error in $.settings.featureFlags.classifiedDomains.config: key \"domains\" not found")
```

According to the current code, only the following is valid:

```yaml
    classifiedDomains:
      status: disabled
      config:
        domains: []
```

two options: change this default; or make the code lenient in parsing such that if the status is disabled, the config object is ignored if absent.

Reply:
Thanks for spotting this! I fixed this in an upcoming commit.